tests/components/climate/test_mqtt.py

```diff
@@ -541,3 +543,17 @@ def test_set_with_templates(self):
         self.hass.block_till_done()
         state = self.hass.states.get(ENTITY_CLIMATE)
         self.assertEqual(74656, state.attributes.get('current_temperature'))
+
+    def test_min_temp_default(self):
+        """Test the default min temp."""
+        assert setup_component(self.hass, climate.DOMAIN, DEFAULT_CONFIG)
+
+        state = self.hass.states.get(ENTITY_CLIMATE)
+        self.assertEqual(7, state.attributes.get('min_temp'))
```

Comment:
So these test cases are not really testing much. The tests on lines 56-57 already do test the defaults. It would, however be good that the configuration schema would be tested with something like

```python
assert setup_component(self.hass, climate.DOMAIN, {
  # ...
  'min_temp': 10.0,
})

state = self.hass.states.get(ENTITY_CLIMATE)
self.assertEqual(10.0, state.attributes.get('min_temp'))
```